controllers/bpfman-agent/ns_xdp_program.go

```diff
@@ -162,9 +160,10 @@ func (r *NsXdpProgramReconciler) updateLinks(ctx context.Context, isBeingDeleted
 func (r *NsXdpProgramReconciler) findLink(attachInfoState bpfmaniov1alpha1.XdpAttachInfoState) *int {
 	for i, a := range r.currentProgramState.XDP.Links {
 		// attachInfoState is the same as a if the the following fields are the
-		// same: InterfaceName, ContainerPid, Priority, and ProceedOn.
-		if a.InterfaceName == attachInfoState.InterfaceName && reflect.DeepEqual(a.ContainerPid, attachInfoState.ContainerPid) &&
-			a.Priority == attachInfoState.Priority && reflect.DeepEqual(a.ProceedOn, attachInfoState.ProceedOn) {
+		// same: IfName, ContainerPid, Priority, and ProceedOn.
```

Comment:
```suggestion
		// same: IfName, NetnsPath, Priority, and ProceedOn.
```

Reply:
This change should happen in my netns pr.

Reply:
Done